### DISCUSSION TITLE
Sysml Distilled DellSat-77 model implemented in Gaphor

### DISCUSSION THREAD

**Discussion by mikekidner** [gaphor/gaphor#3596] [Show and tell]
@danyeaw 
As an internal training tool I've started to implement the SysML Distilled DellSat-77 model in Gaphor.
[Its up on github](https://github.com/mikekidner/gaphor_distilled)
Hopefully it's a good intro to Gaphor (and folks can read the book and the docs along side) and also provides a few good reference diagrams to test the functionality and highlight issues with features.

**Comment by danyeaw** [comment]
This sounds like a good learning resource for sure!
